README.md

```diff
@@ -59,6 +64,11 @@ Then it can be used passing in numeric values for amount and ISO 4217 currency i
 ```php
 $currencyTransformer->toWords(5099, 'USD'); // outputs "fifty dollars ninety nine cents"
 ```
+You can also use the fluent version:
+
+```php
+NumberToWords::currencyTransformer('en')->toWords(5099, 'USD') // outputs "fifty dollars ninety nine cents"

```

Comment:
I think this requires an update according to the latest changes (static methods).

Reply:
Ups, my bad